readme.md

```diff
@@ -16,7 +16,7 @@ INTERCOM_PLUGIN_TEST=1 phpunit
 
 # Test the new version of the plugin with Intercom's Wordpress signup flow
 
-It is mandatory that you fully test the [Intercom wordpress start guide](https://app.intercom.io/a/apps/_/platform/guide/setup_messenger/install_messenger) before you release a new update of the plugin.
+It is mandatory that you fully test the [Intercom wordpress start guide](https://app.intercom.com/a/apps/_/platform/guide/setup_messenger/install_messenger) before you release a new update of the plugin.
```

Comment:
Is this link still active? I get a "We couldn't find the page you're looking for" banner when I visit it.